nucypher/utilities/prometheus/collector.py

```diff
@@ -100,12 +104,6 @@ def initialize(self, metrics_prefix: str, registry: CollectorRegistry) -> None:
             "work_orders_gauge": Gauge(f'{metrics_prefix}_work_orders',
                                        'Number of accepted work orders',
                                        registry=registry),
-            "policies_held_gauge": Gauge(f'{metrics_prefix}_policies_held',
-                                         'Policies held',
-                                         registry=registry),
-            "availability_score_gauge": Gauge(f'{metrics_prefix}_availability_score',
```

Comment:
Because this is removed, it needs to be removed from collection logic, L124-127.